src/status_im/contexts/shell/share/profile/view.cljs

```diff
@@ -30,7 +29,7 @@
         :width               (- window-width (* style/screen-padding 2))
         :qr-data             universal-profile-url
         :qr-data-label-shown abbreviated-url
-        :on-share-press      #(list-selection/open-share {:message universal-profile-url})
```

Comment:
Is `list-selection/open-share` used anywhere else anymore? If not, should be fine to remove

Reply:
That function is used in other places inside the legacy codebase, though I don't think some of those screens are accessible anymore. I opted to leave the legacy code alone, but I can update those call sites to the new event if needed. Thoughts?

Reply:
I guess it's fine leaving it as is then :+1: